Octokit.Tests/Models/IssueTest.cs

```diff
@@ -257,7 +257,7 @@ public void CreatesAnIssueUpdateRequestObject()
 
             var update = issue.ToUpdate();
 
-            Assert.Null(update.Labels);
+            Assert.NotNull(update.Labels);
```

Comment:
Yeah that's a bit odd (IMO!).

I'd expect `.ToUpdate()` to give me a "perfect" replica of the issue, including Labels/Assignees/whatever that I can then use to update the `Issue`.  

I see what you're saying about prefering `AddLabel()` and `ClearLabels()` to direct list manipulation, but if you consider the use case of where I want to add a label to something, I cant just `ToUpdate()` and then `.AddLabel(newLabel)` because that would remove all the other labels.  So I would need more code to explicitly set the labels to whatever they were originally (which perhaps if im now in some other function I dont even have the original `Issue` object anymore, I only have the `IssueUpdate` I'd passed in?  It just seems like this behaviour is a bit odd and means `ToUpdate()` didnt actually create me a "usable" `IssueUpdate` request for my intended usage.  Conversely if we DID include the exiting labels, and possibly fleshed things out with a `RemoveLabel()` call as well as the `AddLabel()` and `ClearLabels()` you still have a way to do everything without direct manipulation but also cater for all intended usages?

It also makes me wonder now, if this is the way we currently handle multiple labels on an issue, does that mean the multiple Assignees functionality added in this PR should also take this approach?  Is the main use case REPLACING assignees or ADDING/REMOVING one of multiple assignees?  Id say the fact that the enhancement is here in the first place means multiple assignees are desirable so we wouldnt want to do things that remove other assignees...